src/create.c

```diff
@@ -72,7 +72,7 @@ parse_opt (int key, char *arg, struct argp_state *state)
       break;
 
     case 'f':
-      config_file = crun_context.config_file = argp_mandatory_argument (arg, state);
```

Comment:
Just a doubt shouldn't config file be part of crun_context rather than container struct, I wonder why is this being removed.

Reply:
I've removed it because we already pass that information as part of `libcrun_container_load_from_file` and `libcrun_container_load_from_memory` so there is no need to duplicate this information in another struct maintained outside the libcrun library.

Reply:
ah makes sense to me.